src/System.DirectoryServices/src/System.DirectoryServices.csproj

```diff
@@ -17,12 +17,15 @@
     <Compile Include="System\DirectoryServices\ActiveDirectorySecurity.cs" />
     <Compile Include="System\DirectoryServices\AdsVLV.cs" />
     <Compile Include="System\DirectoryServices\AuthenticationTypes.cs" />
-    <Compile Include="System\DirectoryServices\BinaryCompatibility.cs" />
     <Compile Include="System\DirectoryServices\DerefAlias.cs" />
     <Compile Include="System\DirectoryServices\DirectoryEntries.cs" />
-    <Compile Include="System\DirectoryServices\DirectoryEntry.cs" />
+    <Compile Include="System\DirectoryServices\DirectoryEntry.cs">
+      <SubType>Component</SubType>
```

Comment:
This gets automatically added when you open the solution with Visual Studio and the class inherits from System.ComponentModel.Component. If you want I can undo this but it will occur again next time someone opens and builds it with VS.